proposals/boost/boost-maven/boost-maven-plugin/src/it/test-mpRestClient/invoker.properties

```diff
@@ -0,0 +1,2 @@
+# User port 9000 to match mpConfig property
```

Comment:
Not about this file in particular..just the whole PR...For the other technologies we have a version in the dir name..e.g. `test-mpRestClient-1.0`... should we here?

Reply:
added a version number to dir name